tests/logictest/logictest.py

```diff
@@ -281,28 +281,26 @@ def execute(self):
         if callable(getattr(self, "batch_execute")):
             # case batch
             for (file_path, suite_name) in self.statement_files:
-                log.info("Batch execute, suite name:{} in file {}".format(
-                    suite_name, file_path))
+                log.info("Run query with the same session every suite file, suite file path:{}".format(file_path))
```

Comment:
As we are using python 3.10, how about using format strings for better reading, like:

```python
log.info(f"Run query with the same session every suite file, suite file path:{file_path}")
```

Reply:
BTW, it's better to add a space after `:`, which will make it easier for reading.